articles/tokens/access-token.md

```diff
@@ -21,6 +21,10 @@ The [Auth0 Management API v1](/api/management/v1) (which has been deprecated) us
 
 Access tokens are issued via Auth0's OAuth 2.0 endpoints: [/authorize](/api/authentication#authorize-client) and [/oauth/token](/api/authentication#get-token). You can use any OAuth 2.0-compatible library to obtain access tokens. If you do not already have a preferred OAuth 2.0 library, Auth0 provides libraries for many languages and frameworks that work seamlessly with our endpoints.
 
+:::note
+See the [Authentication API Explorer](/api/authentication#get-token) for additional usage notes on obtaining access tokens. 
```

Comment:
This link already exists a couple of lines above (`[/oauth/token](/api/authentication#get-token)`). 

I think we should link at the api-auth docs. They have more background info. You could list all of them, for example:
- For an example on how to retrieve an access_token from a server-side web app, refer to [LINK](https://auth0.com/docs/api-auth/tutorials/authorization-code-grant)
- For an example on mobile apps....
- and so forth

Just enhance the list of bullets below this, no need for a note. You could group these options under a `Using our Authentication API` bullet.